### ISSUE TITLE
App View Page

### ISSUE THREAD

**Issue by yaatehr** [UA-Tech-MIT/StudentAppStore#19]
App view page
features - Name, Author, Image, Rating, Description, like button, contact us, link to real page (chrome store extension, app store for mobile apps, etc)
Review displays and review submission text editor
Toggleable edit view for app owner
Add tags?
View counter/ link to resource has been clicked x many times

**Comment by mmenkiti** [comment]
Hey Yaateh, what do you mean by toggleable edit view for app owner? Like should there be a toggle for the owner to edit the page, like the app description and stuff like that? And would that require a login or something so that only owners can edit content?

**Comment by yaatehr** [comment]
Yes exactly. You’ve seen it before. It’s just a page with an edit and save button. That’s togglig the edit view
